arch/sim/Kconfig

```diff
@@ -504,6 +504,21 @@ config SIM_HCISOCKET
 		control of the device, but is abstracted from the
 		physical interface which is still handled by Linux.
 
+config SIM_LINUXI2CBUS
+	bool "Attach Linux I2C Bus"
+	default false
```

Comment:
don't need

Reply:
Don't I still need a flag to control turning on the linux i2c bus implementation? It just would not be used in code in `boards/sim/sim/src/`

The board code would be something like
`SIM_I2C` which would depend on (`SIM_LINUX_I2CBUS` || `SIM_WIN_I2CBUS`)
This would conditional out `sim_i2c_initialize(bus)` logic.

Reply:
Ok I see that is basically what you wrote in another comment.  Sounds like we are on the same page.